api/mail/__init__.py

```diff
@@ -1,2 +1,2 @@
+from .message import DeletedEMail, GZippedFOFN, StagedEMail, WarnedEMail
```

Comment:
I suspect it would be a good idea to create a namespace surrounding all these emails, i.e.
```python
class EmailNamesapce(T.SimpleNamespace):
    Deleted = DeletedEmail
    ...
```
so we can do `from api.mail import EmailNamespace as Email` and then call stuff as `Email.Deleted` for example